src/main.cpp

```diff
@@ -894,12 +950,11 @@ void configModeCallback(WiFiManager *myWiFiManager)
       centeredText("(with mobile data turned off):", DISPLAY_RESOLUTION_X / 2, 110);
       centeredText("1) Connect to this AP:", DISPLAY_RESOLUTION_X / 4, 140);
       centeredText("2) Open in web browser:", DISPLAY_RESOLUTION_X * 3 / 4, 140);
-
-      drawQrCode(qrString.c_str(), 4, 225, DISPLAY_RESOLUTION_X / 4 + 18, 3);
+      int qrScale = (DISPLAY_RESOLUTION_Y < 280) ? 2 : 3;
```

Comment:
I know that's the primitive type, but as it's a compile time, you can use `const`.
// multiple places